tutorial/20_recipes/012_artifact_tutorial.py

```diff
@@ -132,12 +133,11 @@ def objective(trial: optuna.Trial) -> float:
 
     study = optuna.create_study(study_name="test_study", storage="sqlite:///example.db")
     study.optimize(objective, n_trials=100)
-    # Loading and displaying artifacts associated with the best trial.
-    best_artifact_id = study.best_trial.user_attrs.get("artifact_id")
-    with artifact_store.open_reader(best_artifact_id) as f:
-        content = f.read().decode("utf-8")
 
-    print(content)
+    # Downloading artifacts associated with the best trial.
+    best_artifact_id = study.best_trial.user_attrs.get("artifact_id")
+    download_file_path = ...  # Set the path to save the downloaded artifact.
+    download_artifact(artifact_store, best_artifact_id, download_file_path)
```

Comment:
```suggestion
    download_artifact(artifact_store, best_artifact_id, download_file_path)
    with open(download_file_path, "r") as f:
        content = f.read().decode("utf-8")
    print(content)
```

Reply:
Your proposed code seems to open `download_file_path` in text mode. Could you explain why `.decode("utf-8")` is necessary?

Reply:
Oh sorry, you are right.
`mode` was supposed to be `rb`, but not `r`. 
I modified the suggestions.

Reply:
I see. I have another question. Do you think it's necessary to call `print(content)` in this tutorial? In my opinion, we don't need to stick to keeping the previous code. Avoiding the call to `print(content)` makes the tutorial simpler and more readable.

What do you think?

Reply:
In my honest opinion, it is better to have these lines for the sanity check as tutorials are, in general, like a hello-world for the target feature.
Yet I agree that we do not need these lines if the readers completely understand what they are doing in each line.

Reply:
I agree with @nabenabe0928's comment to add the code to open the file. I think that if the tutorial code ends with `download_artifact`, users need to read the API reference to understand how to use the downloaded artifact.